show/main.py

```diff
@@ -1000,7 +1056,7 @@ def ipv6():
 @ipv6.command()
 def interfaces():
     """Show interfaces IPv6 address"""
-    header = ['Interface', 'IPv6 address/mask', 'Admin/Oper']
+    header = ['Interface', 'Master', 'IPv6 address/mask', 'Admin/Oper']
```

Comment:
Can we have 'VRF' header instead of 'Master'?

Reply:
I think 'Master' may be more precise, while device name contains Vrf prefix either.